markdown.dtx

```diff
@@ -25329,6 +25649,73 @@ end
 %    \end{macrocode}
 % \begin{markdown}
 %
+%#### Tex Math Dollars
+%
+% The \luamdef{extensions.tex_math_dollars} function implements the Pandoc
+% tex_math_dollars syntax extension.
+%
+% \end{markdown}
+%  \begin{macrocode}
+M.extensions.tex_math_dollars = function()
+  return {
+    name = "built-in tex_math_dollars syntax extension",
+    extend_writer = function(self)
+%    \end{macrocode}
+% \par
+% \begin{markdown}
+%
+% Define \luamdef{writer->tex_math_dollars} as a function that will transform
+% a tex_math_dollars span `s` of input text to the output format.
```

Comment:
No function `writer->tex_math_dollars` is defined and the functions `writer->display_math` and `inline_math` that are defined below are undocumented.